client/src/less/billing.less

```diff
@@ -0,0 +1,59 @@
+@primary: #cc0000;
```

Comment:
Very nice LESS. We ought to set up an `index` or `main` file so we can bring all the styles in at a single entry point, as well as a `vars` file so we don't have to keep redefining the color scheme.

Reply:
Yeah I've been thinking the same thing, however I didn't want to be the one to start it and mess up some other code at that point, I used the same classes about three times also. There's quite a bit of reusable code in what I did.